test/clj/puppetlabs/trapperkeeper/services/webserver/normalized_uri_helpers_test.clj

```diff
@@ -116,8 +116,8 @@
     ;;> or the surrogate pair ED A1 8C ED BE B4 into U+233B4.  Decoding
     ;;> invalid sequences may have security consequences or cause other
     ;;> problems.
-    (is (= "��" (normalize-uri-path-for-string "%C0%AE")))
-    (is (= "/foo/��/��" (normalize-uri-path-for-string "/foo/%C0%AE/%C0%AE")))))
+    (is (= "-64-82" (normalize-uri-path-for-string "%C0%AE")))
```

Comment:
decodePath began catching NotUtf8Exceptions and handling them by falling back to ISO88591 (https://github.com/eclipse/jetty.project/commit/7f62f2600b943b9aed0e4771891939bf61372c5a#diff-3f826383561a9af15d610a1aa21187d9R336).